src/theme/components/template.ts

```diff
@@ -1,111 +1,84 @@
-// From @mixin breakout:
-// Used for elements that should be edge-to-edge.
+/**
+ * Grid layout based on https://www.joshwcomeau.com/css/full-bleed/
+ */
+
 const breakout = {
-  marginEnd: "calc(-50vw + 50%)",
-  marginStart: "calc(-50vw + 50%)",
-};
-// From the `wrapper` SCSS mixin.
-// TODO: Eventually we may be able to put shared JS style objects
-// into a utils file for other mixins that are shared.
-const wrapperStyles = {
-  marginY: 0,
-  marginX: "auto",
-  maxWidth: "1280px",
-  paddingTop: 0,
-  paddingBottom: 0,
-  paddingEnd: "s",
-  paddingStart: "s",
   width: "100%",
+  // This could be "1 / 4" and it would mean the same. This is
+  // "future-proof" the grid column assignment to the last column.
+  gridColumn: "1 / -1",
```

Comment:
This is no longer working for all the header elements.  The styles probably need to be added to the new `<Box>` wrapper.  If that is the issues, that fix will work in a prop controlled implementation of `Template`, but this may be a problem for using the template children.  How do we ensure that the breakout styles are applied to that wrapper elements?

Reply:
Made a separate PR to work on this. Let me know what you think. #1109